osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -716,30 +745,27 @@ BOOL setEnvVars() {
         return FALSE;
     }
     
-    
-    // Path of the KALITE_PYTHON environment variable
-    NSString* envKalitePythonStr = getEnvVar(@"KALITE_PYTHON");
-    if (! pathExists(envKalitePythonStr)) {
-        return FALSE;
-    }
     NSString *KaliteHomeStr = [NSString stringWithFormat:@"%@",
                                [NSString stringWithFormat:@"launchctl setenv KALITE_HOME \"%@\"", kaliteHomePath]
                                ];
-    NSString *KalitePythonStr = [NSString stringWithFormat:@"%@",
-                                 [NSString stringWithFormat:@"launchctl setenv KALITE_PYTHON \"%@\"", envKalitePythonStr]
-                                 ];
     
-    NSString *org = @"org.learningequality.kalite";
+    // Add prefs to the KALITE_HOME plist name because we have also the same name with KALITE_PYTHON plist at root LaunchAgents.
```

Comment:
What do you mean by "root LaunchAgents"?  Can you specify the name of the existing .plist for reference?